pom.xml

```diff
@@ -23,7 +23,7 @@
         <karaf.version>4.3.6</karaf.version>
         <log4j.version>2.18.0</log4j.version>
         <mockito.version>2.18.0</mockito.version>
-        <opennms.api.version>1.0.1</opennms.api.version>
+        <opennms.api.version>1.1.0</opennms.api.version>
```

Comment:
I wonder if we are ok to have here a fixed version? Can we install the plugin on a higher Api version? I have never tried that...

Reply:
this is expected OIA version for H31. I do test it manually yesterday looks fine.